components/ContractStatus/ContractStatus.tsx

```diff
@@ -15,7 +15,7 @@ export function ContractStatus() {
     error: pricesDataError,
   } = useControllerPricesData();
   const contractAPR = useMemo(() => {
-    if (!pricesData) {
+    if (!pricesData || pricesData.targetValues.length < 2) {
```

Comment:
There are now cases where we briefly have an empty array of targets, handled those in this PR as they came up